src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils/index.ts

```diff
@@ -63,7 +63,7 @@ export const isBoolean = (type: string): boolean => type.indexOf('bool') === 0
 export const isString = (type: string): boolean => type.indexOf('string') === 0
 export const isUint = (type: string): boolean => type.indexOf('uint') === 0
 export const isInt = (type: string): boolean => type.indexOf('int') === 0
-export const isByte = (type: string): boolean => type.indexOf('byte') === 0
+export const isBytes = (type: string): boolean => type.indexOf('bytes') === 0
```

Comment:
Is `"byte"` never returned?

Reply:
I was checking Solidity documentation and all the byte arrays types start with `bytes...`. 
Curiously also found that
> Prior to version 0.8.0, byte used to be an alias for bytes1.

I think we are safe this way but will revert the expression as it is not adding much value.

Reply:
I'd suggest renaming the function back then.